web/src/containers/Request.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react';
```

Comment:
this seems like it may be better off as a functional component that utilizes the `useState` hook

Reply:
tried a bit on integrating hooks with action creator but no luck, will work on this transformation in another pr.

Reply:
https://github.com/bcgov/realm-o-matic/issues/22